src/app/lib/logger.node.js

```diff
@@ -20,19 +20,18 @@ const logLocation = path.join(LOG_DIR, LOG_FILE);
 // prettier-ignore
 const fileTransport = new (transports.File)({
   filename: logLocation,
-  handleExceptions: true,
-  humanReadableUnhandledException: true,
+  handleExceptions: false,
```

Comment:
Now we've increased the listeners, why do we need to suppress the exceptions; I'd have thought an increase in listeners would prevent the errors?

Reply:
After increasing listeners, there was still 1 warning left:
`(node:21424) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 uncaughtException listeners added to [process]. Use emitter.setMaxListeners() to increase limit`

All of the others were file exceptions (not process). Setting `handleExceptions` to `false` seemed to be the only way to get rid of that warning 😢 

Are we happy enough with this approach?

Reply:
Kinda feel like one day we could end up with logs going missing and this error could be the 'smoking gun' that we may be choosing to ignore. Is it worth another quick dig into this, I'm happy to help?

That being said, I don't want to block this, if others are happy, we can merge 🙂

Reply:
Yes, let's have another look at this to see if we can come up with a better solution.